pkg/model/estimator/local/benchmark_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local_test
+
+import (
+	"strconv"
+	"testing"
+
+	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
+	"github.com/sustainable-computing-io/kepler/pkg/config"
+	"github.com/sustainable-computing-io/kepler/pkg/model/estimator/local"
+	"github.com/sustainable-computing-io/kepler/pkg/power/components/source"
+)
+
+func benchmarkNtesting(b *testing.B, continerNumber int) {
+	containersMetrics := map[string]*collector_metric.ContainerMetrics{}
+	nodeMetrics := collector_metric.NewNodeMetrics()
+	collector_metric.ContainerMetricNames = []string{config.CoreUsageMetric}
+	nodeMetrics.AddNodeResUsageFromContainerResUsage(containersMetrics)
```

Comment:
is this needed? seems containersMetrics  is empty at this point?

Reply:
adjust this part after data creation.